panels/templates/panel_app_form.html

```diff
@@ -9,6 +9,9 @@
             isNotAnHour = $.val('length') != {{ c.SIXTY_MIN }};
         setVisible($('#other-length'), isOther);
         $.field('length_text').prop('required', isOther);
+        if (!isOther) {
```

Comment:
That's true! I didn't even think of that!

The thing I was trying to prevent is the following:
1. Someone chooses "Other"
2. Then they type 3 hours into the length
3. Then they change their mind and choose "2 hours"
4. Both fields get saved when they submit the form
5. Now the backend will display "3 hours" when it really should be "2 hours" 😦